### PR TITLE
Privacy Policy

### PR DESCRIPTION
**Pull request by alans-42**
#47

### CODE REVIEW

**Comment by alans-42** [comment]
Generator used: https://app-privacy-policy-generator.nisrulz.com

File: authenticator/App.js [hunk 1]
```diff
@@ -3,19 +3,20 @@ import React from 'react';
 import { NavigationContainer } from '@react-navigation/native';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
 import { StyleSheet, View } from 'react-native';
+import Toast from 'react-native-toast-message';
 import ApplicationList from './views/application-list';
 import Settings from './views/settings';
 import New from './views/new';
 import Confirm from './views/confirm';
 import License from './views/license';
+import Privacy from './views/privacy-policy';
```
Comment (emilkovacev) [inline authenticator/App.js:12]: I believe there is no such file at the moment. We can probably just paste the html that we had from that generator and format it with similar js as before!

**Review by emilkovacev — CHANGES_REQUESTED** [review]
Admittedly it took me some time before I managed to get to this review, but here it is! Just one small change that seems to be preventing the app from running. Once that's done we can look at the functionality and make that pr!

**Comment by alans-42** [comment]
Sorry I just noticed the comment, but I realized the file with the policy itself was never committed and pushed so I just added it to the pull request.

**Comment by emilkovacev** [comment]
Thanks for updating the PR! There seem to still be some small bugs I'd like to iron out, but honestly it may be on my end. I'll keep this chat up to date on the progress for the PR, but it should be merged soon :)

**Pull request closed without merging**